week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -55,15 +74,32 @@ function exerciseTwo(shopping) {
     - Add an <img> to each book that links to a URL of the book cover.
     - Change the style of the book depending on whether you have read it (green) or not (red).
 
+    
+
     The end result should look something like this: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com/
 **/
-function exerciseThree(books) {
-  //Write your code in here
+function exerciseThree(books){
+let content = document.getElementById("content");
```

Comment:
Looks like this function is missing indentation and appropriate spacing.
Indentation is quite important as it makes your code a lot easier to read and understand 😉

Check this out: https://javascript.info/coding-style